CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ $ docker compose up
 1. Installing the Ollama model
 
    ```
-   $ docker exec -it ollama ollama run llama3
+   $ docker exec ollama ollama pull llama3
```

Comment:
extra: this is an easier way (IMO) to pull it so that the container exits, it's less confusing

Reply:
Maybe we could improve this by creating a makefile or a shell script for mac and linux/bat file for windows. IMHO it's more convenient to run a single command that sets up and configures everything for you, including downloading the llama3 LLM